app/models/manifest_uploader.rb

```diff
@@ -47,27 +47,7 @@ def locations
   end
 
   def missing_locations
-    @missing_locations ||= location_barcodes.reject { |barcode| locations.key?(barcode.strip) }
-  end
-
-  def ordered_location_barcodes
-    @ordered_location_barcodes ||= location_barcodes.select { |barcode| locations[barcode.strip].type == "OrderedLocation" if locations.include?(barcode.strip) }
-  end
-
-  def ordered_location_rows
-    @ordered_location_rows ||= find_ordered_location_rows
-  end
-
-  def find_ordered_location_rows
-    rows = []
-    data.each_with_index do |row, index|
-      location_barcode = row.first.strip
-      if ordered_location_barcodes.include?(location_barcode)
-        indexed_row = [(index + 2).to_s] + row
-        rows.push(indexed_row)
-      end
-    end
-    rows
+    @missing_locations ||= location_barcodes.reject { |barcode| locations.key?(barcode) }
```

Comment:
This error message shows even if the location is blank (empty)
Shows: location(s) with barcode do not exist 
Maybe this should be more specific saying an uploaded barcode is blank